code/modules/atmospherics/machinery/components/binary_devices/passive_gate.dm

```diff
@@ -13,6 +13,15 @@
 
 	var/id = null
 
+/obj/machinery/atmospherics/binary/volume_pump/can_be_pulled(user, grab_state, force, show_message)
+	return FALSE
```

Comment:
what if it is unwrenched?

Reply:
Then it just becomes a normal item pipe, so you can drag it